spring-pulsar-spring-cloud-stream-binder/src/main/java/org/springframework/pulsar/spring/cloud/stream/binder/properties/PulsarBinderConfigurationProperties.java

```diff
@@ -36,7 +36,7 @@ public class PulsarBinderConfigurationProperties {
 	@NestedConfigurationProperty
 	private final ProducerConfigProperties producer = new ProducerConfigProperties();
 
-	private int partitionCount = 1;
+	private int partitionCount = 0;
 
```

Comment:
Not directly related to this PR, but Do you mind changing the accessor from `partitionCount` to `getPartitionCount` to keep consistent w/ the other config props.